src/invidious/yt_backend/youtube_api.cr

```diff
@@ -610,6 +665,8 @@ module YoutubeAPI
       "x-goog-api-format-version" => "2",
       "x-youtube-client-name"     => client_config.name_proto,
       "x-youtube-client-version"  => client_config.version,
+      "accept"                    => "*",
+      "accept-language"           => "*",
```

Comment:
U suspect these needs to coincide with what's already set in `youtube_pool.cr`

Reply:
good point, I have switched `accept` to `*` like on the official youtube.com website.